interfaces/builtin/u2f_devices.go

```diff
@@ -138,9 +138,9 @@ var u2fDevices = []u2fDevice{
 		ProductIDPattern: "a2ac",
 	},
 	{
-		Name:             "Ledger Blue + Nano S + Nano X",
+		Name:             "Ledger Blue + Nano S + Nano X + Nano S Plus",
 		VendorIDPattern:  "2c97",
-		ProductIDPattern: "0000|0001|0004|0005|0015|1005|1015|4005|4015",
+		ProductIDPattern: "0000|0001|0004|0005|0015|1005|1015|4005|4015|5011",
```

Comment:
Just one question, this pci-id is under the `Ledger Test` comment in the upstream(?) repo https://github.com/LedgerHQ/udev-rules/blob/master/add_udev_rules.sh#L15 it seems to not matter but I wonder why?

Reply:
I presumed myself that they haven't updated the current udev rules since the nano s plus was still being developed. I just opened an issue with them, requesting clarification here:

https://github.com/LedgerHQ/udev-rules/issues/20

Reply:
> Just one question, this pci-id is under the `Ledger Test` comment in the upstream(?) repo https://github.com/LedgerHQ/udev-rules/blob/master/add_udev_rules.sh#L15 it seems to not matter but I wonder why?

I signed up with the ledger discord chat server, on advice of ledger support staff, and have finally got a response from a developer who works for ledger. They are looking into who they need to contact on their team, to help in responding to my inquiry with them, at their udev-rules repository (link above in previous reply).

Reply:
@mvo5, I presume they may never get back to us for comment...see discord screenshot. No updates from them for 2 weeks. Let me know if there is anything else I can do, I have the Nano S Plus to test with.
![Workspace 1_017](https://user-images.githubusercontent.com/1760489/207623831-6e8fc9cf-d6bb-4498-a038-d78ceb7eb77b.png)

Reply:
Thanks for chasing this ! A bit sad that there is no update to the udev rules in the upstream git repo, I will ask in one of their issues too,  I would just feel better if it was clearly documented what the usb id is

Reply:
No problem, my pleasure. Agreed the docs should be clearer / explained by the hardware devs. Maybe you'll have better luck than me. Holler if you need anything.